modules/lz4/CMakeLists.txt

```
@@ -0,0 +1,20 @@
+# Copyright (c) 2020 Linumiz
+# SPDX-License-Identifier: Apache-2.0
+
+if(CONFIG_LZ4)
+
+  set(LZ4_DIR ${ZEPHYR_CURRENT_MODULE_DIR})
+
+  set(CMAKE_MODULE_PATH ${CMAKE_MODULE_PATH} ${LZ4_DIR}/extra)
+  find_package(lz4 REQUIRED)
+
+  zephyr_library()
+
+  zephyr_include_directories(${LZ4_DIR})
+
+  zephyr_library_sources(
+    ${LZ4_DIR}/lz4.c
+    ${LZ4_DIR}/lz4hc.c
```

also missing `lib` here:

then it seems there is a missing `lib` here.

```suggestion
    ${LZ4_DIR}/lib/lz4.c
    ${LZ4_DIR}/lib/lz4hc.c
```